### PR TITLE
Thumbnails In Search Results (No Thumbnail When Playing)

### PR DESCRIPTION
**Pull request by artforlife**
The search results are now displayed with thumbnails. No image is displayed when a video is selected for playing.
### Important

Mark with [x] to select. Leave as [ ] to unselect.
### Motivation and Context
- [ ] Why is this change required? What problem does it solve?
- [ ] If it fixes an open issue, include the text `Closes #1` (where 1 would be the issue number) to your commit message.
### Types of changes

What types of changes does your code introduce? Check all the boxes that apply:
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
### Description
- [x] Describe your changes in detail.
### Final checklist:

Go over all the following points and check all the boxes that apply  
If you're unsure about any of these, don't hesitate to ask. We're here to help!
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING**](.github/CONTRIBUTING.md) guidelines.
- [x] All tests passed.

### CODE REVIEW

File: css/styles.css [hunk 1]
```diff
@@ -137,8 +137,20 @@ a {
...
 }
 
+#search-results a img {
+    vertical-align: middle;
+    border-radius: 8px;
+    margin-right: 5px;
+    padding: 2px;
+}
+
+
```
Comment (shakeelmohamed) [inline css/styles.css]: nit: extra blank line

File: js/everything.js [hunk 2]
```diff
@@ -495,11 +495,11 @@ $(function() {
                     $("#search-results").show();
                     // Clear out results
                     $("#search-results ul").html("");
-
-                    var start = "<li><h4><a href=?v=";
+                    var start = "<li><h4><a style='float:right' href=?v=";
```
Comment (shakeelmohamed) [inline js/everything.js]: I don't like using single quotes in HTML, let's update all of these

File: js/everything.js [hunk 3]
```diff
@@ -495,11 +495,11 @@ $(function() {
...
                     var end = "</a></h4></li>";
                     $.each(data.items, function(index, result) {
-                        $("#search-results ul").append(start + result.id.videoId + ">" + result.snippet.title  + end);
+                        var thumbTag = "<img style='float:left; margin: auto' src='https://img.youtube.com/vi/" + result.id.videoId + "/1.jpg'>";
```
Comment (shakeelmohamed) [inline js/everything.js]: I'd like to break this up into a couple of steps, make the image URL it's own var, etc to make this more readable.

File: js/everything.js (hunk 2 above)
Comment (shakeelmohamed) [inline js/everything.js]: Let's remove all the inline style and create a new CSS class, same for the results themselves

**Comment by shakeelmohamed** [comment]
Can you take a screenshot with before/after?

**Comment by artforlife** [comment]
Here is a screenshot. I shall clean up the code soon.
screen shot 2016-07-30 at 9 40 35 am

**Comment by shakeelmohamed** [comment]
Thanks, it definitely disturbs the current layout. We might want to use smaller thumbnail images (there may be a different URL for different sizes of the same thumbnail), and probably left-align the video title so it looks more like YouTube - see the screenshot below:

capturfiles_2

**Comment by artforlife** [comment]
I can left-align the title without a problem. How much smaller do we want the thumbnails? They are already quite tiny. Also, I do not see how they disturb the layout unless I simply misunderstand what you mean.

**Comment by shakeelmohamed** [comment]
> How much smaller do we want the thumbnails?

Let's try 50% of how they are now, can we make sure the aspect ratio is correct as well? For ABGT200 (second result in both screenshots), in your implementation there are horizontal black bars unlike the YouTube search results.

> Also, I do not see how they disturb the layout unless I simply misunderstand what you mean.

The video titles are center aligned, but also pushed to the right by X pixels (where X is the width of the thumbnail).

**Comment by artforlife** [comment]
I was away for awhile. Will do a PR in next couple days. Also back on the Electron side!

**Comment by shakeelmohamed** [comment]
@artforlife thanks for the update!

**Comment by shakeelmohamed** [comment]
Closing due to inactivity

**Pull request closed without merging**